main.js

```diff
@@ -1,20 +1,23 @@
 $(document).ready(function() {
-    var postIt = $(".sticker");
+    // var postIt = $('.sticker');
```

Comment:
Remove commented code everywhere, use comments in code only if it's required to clarify code.